### PR TITLE
Issue #148 Adding git commit message validation

### PR DESCRIPTION
**Pull request by hferentschik**

### CODE REVIEW

**Comment by joshuawilson** [comment]
don't forget to update the README so others know what you expect

File: Makefile [hunk 1]
```diff
@@ -89,6 +92,10 @@ clean: ## Deletes all build artifacts
 	rm -rf tools.timestamp
 	rm -rf $(BUILD_DIR)
 
+.PHONY: validate_commits
+validate_commits: tools ## Validates git commit messages
+	git-validation -q -run short-subject,message_regexp='^Issue #[0-9]+  ?[A-Z]+.*' -range $(START_COMMIT_MESSAGE_VALIDATION)...
```
Comment (vpavlin) [inline Makefile]: Are you saying there can be 1 or 2 spaces between the number and text? Why 1 or 2?

Comment (hferentschik) [reply Makefile]: Well, actually no, but I need a start-commit and the previous commit had two spaces. The plan was to change this in a later commit. But I have a better idea now. I will add a commit before this one updating the README. Then this commit can serve as starting point.

**Comment by hferentschik** [comment]
> don't forget to update the README so others know what you expect

will do, even though the message will be pretty obvious.

File: Makefile (hunk 1 above)
Comment (hferentschik) [reply Makefile]: Trying to put some safeguards in place for the time to come. ;-)

Comment (vpavlin) [reply Makefile]: I completely understand the reasoning, just the 2 spaces confused me:D

**Comment by vpavlin** [comment]
LGTM

**Comment by hferentschik** [comment]
cheers

**Pull request merged**